core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -3558,6 +3558,7 @@ predicate
     | expression comparisonOperator (ALL | SOME | ANY) L_PAREN subquery R_PAREN                  # predASA
     | expression IS NOT? NULL                                                                    # predIsNull
     | expression NOT? IN L_PAREN (subquery | exprList) R_PAREN                                   # predIn
+    | L_PAREN exprList R_PAREN  NOT? IN L_PAREN (subquery | exprList) R_PAREN                    # predExprListIn
```

Comment:
Looks like the ANTLR formatter has been disabled. Extra space crept in there.

Reply:
done